.github/workflows/programs-cryptid-signer.yml

```diff
@@ -130,3 +130,59 @@ jobs:
         env:
           RUST_LOG: solana_rbpf::vm=debug,solana_runtime::message_processor=debug,solana_runtime::system_instruction_processor=trace,solana_program_test=info,cryptid_signer=trace
           RUST_BACKTRACE: 1
+
+  program-dev-deploy:
+    name: Deploy dev cryptid on-chain program, built on Solana ${{ matrix.solana }}, Rust ${{ matrix.rust }}, and ${{ matrix.os }}
+    if: github.ref == 'refs/heads/develop' && github.event == 'push'
+    needs: [program-lint, program-unit-test, program-integration-test]
+    env:
+      SOLANA_PROGRAM_ID: tcrjc6mfci8bQLmXPfrVw9fJm9Y5tr268tByjSQmSe9
+      SOLANA_AUTHORITY: ${{ secrets.SOLANA_DEV_AUTHORITY }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        rust: ['1.55']
+        solana: ['v1.7.12']
+        os: [ubuntu-latest]
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Use Rust ${{ matrix.rust }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+
+      - name: Cache build dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            ./target
+          key: cargo-build-${{ hashFiles('Cargo.lock') }}
+
+      - name: Cache Solana version
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache
+          key: solana-${{ matrix.solana }}
+
+      - name: Install Solana
+        run: |
+          sh -c "$(curl -sSfL https://release.solana.com/${{ matrix.solana }}/install)"
+          echo "$HOME/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
+
+      - name: Build programs
+        uses: actions-rs/cargo@v1
+        with:
+          command: build-bpf
+
+      - name: Deploy dev program
+        run: |
+          echo "$SOLANA_AUTHORITY" > ~/.config/solana/id.json
```

Comment:
is both line 185 and the echo in 187 needed? I thought it would use id.json by default if not passed as --upgrade-authority?

Reply:
You are correct. I removed the one on line 187